app/assets/javascripts/charts.js

```diff
@@ -65,9 +67,15 @@ function expanded_pie(target, data){
         show: true,
         innerRadius: 0.8*3/4,
         radius: 0.8,
-        labels: {
```

Comment:
Hmm, is this either/or?  So we have either one large label in the middle *or* the labels on the outside?  I just tried leaving both labels + label in, but only the label showed still.

I think it would be nice if possible to keep both, since in the fact charts we have enough room for both.  In the dashboard we only really have room for one.